### PR TITLE
Update changed_organization.html

### LINKED ISSUE #5771: Organization changes do not have detailed Activity Stream UI
**CKAN version**
2.9.1

**Describe the bug**
One of the major features in 2.9 is the detailed Dataset Activity Stream.  Even though Organization changes are also stored in the `activity` table, it only logs that the Organization has been updated.

**Steps to reproduce**
Make changes to an Organization and check the Activity Stream.

**Expected behavior**
The Activity Stream should at least show the diff, similar to how its done with Datasets, as the Organization metadata is stored in `activity.data` column.

### PR DESCRIPTION
**Pull request by Gauravp-NEC**
Fix for Issue 5771

Fixes #5771 

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by Gauravp-NEC** [comment]
@Zharktas , Please review this PR. Thankyou :)

**Comment by Gauravp-NEC** [comment]
@Zharktas , Please review this PR. Thankyou :)

File: ckan/views/group.py [hunk 1]
```diff
@@ -827,6 +827,47 @@ def post(self, id, group_type, is_organization, data=None):
...
+    '''
+    activity_id = id
+    context = {
+        u'model': model, u'session': model.Session,
+       u'user': g.user, u'auth_user_obj': g.userobj
+    }
+    try:
+        activity_diff = get_action(u'activity_diff')(
+            context, {u'id': activity_id, u'object_type': u'group',
+                      u'diff_type': u'html'}
```
Comment (Zharktas) [inline ckan/views/group.py:842]: There is a syntax error which causes ckan not to start in tests.
```suggestion
                      u'diff_type': u'html'})
```

**Review by Zharktas — CHANGES_REQUESTED** [review]

File: ckan/views/group.py (hunk 1 above)
Comment (Gauravp-NEC) [reply ckan/views/group.py]: @Zharktas , I have corrected the syntax error in commit https://github.com/ckan/ckan/pull/5912/commits/50fa7dc62ee48a882791c9f0f32213b218b78652

**Comment by Gauravp-NEC** [comment]
@Zharktas , I have corrected the syntax error in commit https://github.com/ckan/ckan/commit/50fa7dc62ee48a882791c9f0f32213b218b78652. Thankyou

File: ckan/views/group.py [hunk 2]
```diff
@@ -827,6 +827,47 @@ def post(self, id, group_type, is_organization, data=None):
...
+        u'model': model, u'session': model.Session,
+       u'user': g.user, u'auth_user_obj': g.userobj
+    }
+    try:
+        activity_diff = get_action(u'activity_diff')(
+            context, {u'id': activity_id, u'object_type': u'group',
+                      u'diff_type': u'html'})
+    except NotFound as e:
+        log.info(u'Activity not found: {} - {}'.format(str(e), activity_id))
+       return base.abort(404, _(u'Activity not found'))
```
Comment (Zharktas) [inline ckan/views/group.py:845]: The return line is wrongly indentended.
```suggestion
        return base.abort(404, _(u'Activity not found'))
```

**Review by Zharktas — CHANGES_REQUESTED** [review]
You really should run this yourself and not just let CI run it for you.

File: ckan/views/group.py (hunk 2 above)
Comment (Gauravp-NEC) [reply ckan/views/group.py]: @Zharktas , I am sorry for this wrong indentation. I have corrected it in commit https://github.com/ckan/ckan/pull/5912/commits/57a3b50ec37e9a70c8ae145a9905e04a1324d21d

**Comment by Gauravp-NEC** [comment]
@Zharktas , I have corrected the indentation error in commit https://github.com/ckan/ckan/pull/5912/commits/57a3b50ec37e9a70c8ae145a9905e04a1324d21d

**Comment by Gauravp-NEC** [comment]
@Zharktas , I have corrected the indentation error in commit https://github.com/ckan/ckan/commit/57a3b50ec37e9a70c8ae145a9905e04a1324d21d. Please review this PR.

**Comment by Zharktas** [comment]
This code is still broken as you can see here https://app.circleci.com/pipelines/github/ckan/ckan/1354/workflows/423f18ed-2aa8-4b4c-b9e2-6ae96fa536d8/jobs/9057, would you please test it yourself first.

**Comment by Gauravp-NEC** [comment]
@Zharktas , Thankyou. I have updated this PR. Please review.

**Comment by Gauravp-NEC** [comment]
@Zharktas . Now all test cases have passed. Please review this PR.

**Comment by Gauravp-NEC** [comment]
@Zharktas , I have done the changes which are required. Now all the test cases have been passed. Please review this PR.

**Comment by Gauravp-NEC** [comment]
@Zharktas @amercader,  I have done the changes which are required. Now all the test cases have been passed. Please review this PR. Thankyou :)

**Comment by Zharktas** [comment]
This doesn't work at all, it errors immediately after going to the organization changes UI.

image

And this feature would require tests too, similar to current ones in https://github.com/ckan/ckan/blob/master/ckan/tests/lib/test_changes.py

**Comment by Gauravp-NEC** [comment]
@Zharktas , It is working on my environment. Is there need of blueprint ''organization.multiple_changes'' and def multiple_changes() similar to dataset?

**Comment by Zharktas** [comment]
The error comes from here https://github.com/ckan/ckan/pull/5912/files#diff-2c451a87f8875d6948efb84a3ea7dd4dab4791255e707ba1c5fd6995d78c832fR22 and these templates should be under organization directory and not package, as they aren't related to datasets.

**Comment by Gauravp-NEC** [comment]
@Zharktas , Thankyou. I will raise a new PR in which your comments will be incorporated.

**Comment by Gauravp-NEC** [comment]
@Zharktas , I have raise a new PR https://github.com/ckan/ckan/pull/5985 in which I have solve the build error and moved the templates under organizations directory.

**Pull request closed without merging**